support-config/src/main/scala/com/gu/support/config/StripeConfig.scala

```diff
@@ -1,24 +1,47 @@
 package com.gu.support.config
 
-import com.gu.i18n.Currency
+import com.gu.i18n.{Country, Currency}
 import com.gu.i18n.Currency.AUD
 import com.gu.monitoring.SafeLogger
 import com.typesafe.config.Config
 
 
-case class StripeConfig(defaultAccount: StripeAccountConfig, australiaAccount: StripeAccountConfig, version: Option[String] = None)
+case class StripeConfig(defaultAccount: StripeAccountConfig,
+                        australiaAccount: StripeAccountConfig,
+                        unitedStatesAccount: StripeAccountConfig,
+                        version: Option[String])
```

Comment:
to stop intellij doing this change these settings 
![image](https://user-images.githubusercontent.com/7304387/64187506-a57f1300-ce68-11e9-88e6-7a8ea7993d50.png)